playbooks/collect_activity/stats.py

```diff
@@ -1,7 +1,8 @@
 #!/edx/bin/python.edxapp
 # pylint: skip-file
 
-from ConfigParser import ConfigParser
+from argparse import ArgumentParser
+from configparser import ConfigParser
```

Comment:
@pcockwell, this doesn't work on Python 2.7, which is what is currently used on the production Open edX instances running on Ironwood and the `configparser` library is not installed. You can use `six.moves.configparser` to import it in a compatible way.

Reply:
Done in next commit